frontend/src/pages/connectionTypes/manage/EditConnectionTypePage.tsx

```diff
@@ -3,12 +3,24 @@ import { useParams } from 'react-router';
 import ApplicationsPage from '~/pages/ApplicationsPage';
 import { useConnectionType } from '~/concepts/connectionTypes/useConnectionType';
 import { updateConnectionType } from '~/services/connectionTypesService';
+import { ownedByDSC } from '~/concepts/k8s/utils';
 import ManageConnectionTypePage from './ManageConnectionTypePage';
 
 const EditConnectionTypePage: React.FC = () => {
   const { name } = useParams();
   const [existingConnectionType, isLoaded, error] = useConnectionType(name);
 
+  if (existingConnectionType && ownedByDSC(existingConnectionType)) {
+    return (
+      <ApplicationsPage
+        loaded={isLoaded}
+        errorMessage="Unable to edit"
+        loadError={new Error('This connection type is not editable')}
+        empty
+      />
+    );
+  }
```

Comment:
I feel like we should just redirect the user back to `/conectionTypes` instead of displaying this page. The only way they can get here is by direct URL anyways.